### PR TITLE
use kubernetes.io/arch label

### PR DESCRIPTION
**Pull request by adamdyszy**

### CODE REVIEW

**Comment by pgodowski** [comment]
We put this item on hold at least till Monday Nov 30th, unless further notice

**Review by SzymonKowalczyk — CHANGES_REQUESTED** [review]
Do not deliver/merge until final decision

File: deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -485,7 +485,7 @@ spec:
                     requiredDuringSchedulingIgnoredDuringExecution:
                       nodeSelectorTerms:
                         - matchExpressions:
-                            - key: beta.kubernetes.io/arch
```
Comment (arturobrzut) [inline deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: @pgodowski @adamdyszy 
This change drop support for OCP 3.X, and for all k8s less than 1.13 (December 2018)
kubernetes.io/arch was added to k8s in 1.13 Release  December 2018
 * OCP 3.11 use Kubernetes 1.11 (last release 3.X)
 * OCP 4.1 use Kubernetes 1.13 (first release 4.X)

Comment (arturobrzut) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: https://github.com/kubernetes/kubernetes/blob/release-1.12/pkg/kubelet/apis/well_known_labels.go
https://github.com/kubernetes/kubernetes/blob/release-1.13/pkg/kubelet/apis/well_known_labels.go

Comment (pgodowski) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: @arturobrzut on OCP 3.11, do we install LS via OLM, or by direct deployment of the operator?

Comment (pgodowski) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: what is the latest on this? @adamdyszy

Comment (adamdyszy) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: @pgodowski we install without OLM

Comment (pgodowski) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: can we merge then?

Comment (arturobrzut) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: @pgodowski 
we can merge it (if we do not support releases OCP less then 4.1)

@pgodowski Can we confirm it 
Then I will merge it

@SzymonKowalczyk only need to resolve his reviewu

Comment (pgodowski) [reply deploy/olm-catalog/ibm-licensing-operator/1.3.1/ibm-licensing-operator.v1.3.1.clusterserviceversion.yaml:488]: @arturobrzut @adamdyszy please followup - I see references to `beta.kubernetes.io` in few other places including deployment.go

**Pull request closed without merging**